docs/source/models.rst

```diff
@@ -182,11 +189,172 @@ TensorFlow (``tensorflow``)
 
 The ``tensorflow`` model flavor enables logging TensorFlow ``Saved Models`` and loading them back as ``Python Function`` models for inference on pandas DataFrames. Given a directory containing a saved model, you can log the model to MLflow via ``log_saved_model`` and then load the saved model for inference using ``mlflow.pyfunc.load_pyfunc``. For more information, see :py:mod:`mlflow.tensorflow`.
 
+Model Customization
+-------------------
+
+While MLflow's built-in model persistence utilities are convenient for packaging models from various 
+popular ML libraries in MLflow model format, they do not cover every use case. For example, you may 
+want to use a model from an ML library that is not explicitly supported by MLflow's built-in 
+flavors. Alternatively, you may want to package arbitrary inference code and data to create an 
+MLflow model. Fortunately, MLflow provides two solutions that can be used to accomplish these 
+tasks: :ref:`custom-python-models` and :ref:`custom-flavors`.
+
+.. _custom-python-models:
+
+Custom Python Models
+^^^^^^^^^^^^^^^^^^^^
+The :py:mod:`mlflow.pyfunc` module provides :py:func:`save_model() <mlflow.pyfunc.save_model>` and 
+:py:func:`log_model() <mlflow.pyfunc.log_model>` utilities for creating MLflow models with the
+``python_function`` flavor that contain arbitrary user-specified code and *artifact* (file) 
+dependencies. These artifact dependencies may include serialized models produced by any Python ML 
+library.
+
+Because these custom models contain the ``python_function`` flavor, they can be deployed
+to any of MLflow's supported production environments, such as SageMaker, AzureML, or local
+REST endpoints.
+
+The following examples demonstrate how the :py:mod:`mlflow.pyfunc` module can be used to create 
+custom Python models. For additional information about model customization with MLflow's 
+``python_function`` utilities, see the 
```

Comment:
The following examples demonstrate how the :py:mod:`mlflow.pyfunc` module can be used to create 
custom Python models. -> The following examples demonstrate how you can use the :py:mod:`mlflow.pyfunc` module to create custom Python models.

Reply:
Changed to active voice. Thanks!